PINRemoteImage.podspec

```diff
@@ -19,12 +19,22 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, '6.0'
   s.requires_arc = true
+  
+  # Exclude optional FLAnimatedImage module
+  s.default_subspec = 'Core'
```

Comment:
@nap-sam-dean 
1. The default behavior will be to exclude FLAnimatedImage.
2. If you aren't using FLAnimatedImage in your project, running `pod update` should not affect your build.